src/components/PrimaryButton.js

```diff
@@ -1,5 +1,8 @@
 import React, {Component} from 'react';
 import {StyleSheet, Text, TouchableHighlight, View} from 'react-native';
+import { theme } from '../constants';
+
+const { COLORS } = theme;
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file and about the style files.